docs/how-to-setup-wsl.md

```diff
@@ -104,12 +106,12 @@ nvm install --lts
 # OR
 # nvm install <version>
 
-nvm install 14
+nvm install 20.9
 
 # Usage
 # nvm use <version>
 
-nvm use 12
+nvm use 20.9
```

Comment:
Let's remove this because clearly, users are skipping over the fact the `--lts` is a better option. We can point users to nvm usage on the nvm docs.